packages/waku/src/router/client.ts

```diff
@@ -125,16 +136,29 @@ export function useRouter_UNSTABLE() {
         '',
         url,
       );
-      changeRoute(parseRoute(url), { shouldScroll: newPath });
+      changeRoute(parseRoute(url), {
+        shouldScroll: options?.scroll ?? newPath,
+      });
```

Comment:
Maybe this too?
```suggestion
      changeRoute(parseRoute(url), { shouldScroll: options?.scroll ?? newPath });
```

Reply:
the multi line one is what prettier changed both to... I need to fix my local prettier it seems

Reply:
oh, I see.